pkg/aws/client_test.go

```diff
@@ -1,10 +1,14 @@
 package aws_test
```

Comment:
You could also make this package be aws instead so you don't have to open up the BuileCustomRetryer just for the test